.vscode/extensions.json

```diff
@@ -7,7 +7,9 @@
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "jrieken.vscode-tree-sitter-query",
-    "wenkokke.tree-sitter-talon",
-    "usernamehw.commands"
+    "wenkokke.talonfmt-vscode",
+    "usernamehw.commands",
+    "sumneko.lua",
```

Comment:
Why do we need two plugins for lua? We only basically need something that provides the language id.

We could actually do this in Cursorless, but only fore testing? @pokey

Reply:
@AndreasArvidsson i think you may be confusing this file with https://github.com/cursorless-dev/cursorless/blob/main/packages/common/src/extensionDependencies.ts?

Reply:
I am. I didn't even know that we had two of these files. sigh :p

Reply:
Ha yeah no worries; this is the one for host vscode instance when cursorless workspace is open; it's what results in the little pop-up with recommended extensions the first time you clone and open cursorless repo as a contributor

Reply:
@pokey can you clarify which one you are talking about? And what is the other one doing differently? :D

Reply:
This file (`.vscode/extensions.json`) is the one for host vscode instance when cursorless workspace is open; it's what results in the little pop-up with recommended extensions the first time you clone and open cursorless repo as a contributor

The other file ([`extensionDependencies.ts`](https://github.com/cursorless-dev/cursorless/blob/main/packages/common/src/extensionDependencies.ts)) installs extensions in the debug host that are necessary for running tests. We basically only need them if we add Cursorless language scope support for a language whose language id isn't supported by default by VSCode. For this PR we don't need it because we're not adding language scope support for any language